docs/windows.md

```diff
@@ -0,0 +1,187 @@
+is invoked to allocate IPs from the subnet to all local Pods.
+
+### Components that run on Windows
+
+The following components should be configured and run on the Windows Node.
+* [kubernetes components](https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-nodes/)
+* OVS daemons
+* Antrea Agent
+* Kube-proxy
+
+Both Antrea Agent and OVS daemons support to run as process or Windows service.
+
+### Traffic walk
+* ***Intra-node traffic*** and ***Inter-node traffic*** On Windows Node, the
+forwarding path for packets that are between two local Pods or to a Pod on
+another Node is the same as Linux Node. Please refer to the [architecture](/docs/architecture.md).
+
+* ***Pod to external traffic*** Antrea Agent creates OpenFlow rules to perform
+SNAT on the packets sent from local Pod to an external IP or the Nodes' network,
+and the source IP will be rewritten to the Node's IP. Then the packets are
+forwarded to the OVS bridge. IP-Forwarding is enabled on the OVS bridge, and the
+pacets will be sent out using host networking stack.
+
+## Deploying Antrea on Windows Worker Node
+
+### Prerequisite
+* Obtain a Windows Server 2019 license (or higher) in order to configure the
+Windows node that hosts Windows containers. And install the latest windows updates.
+* Build a Linux-based Kubernetes cluster.
+* Install [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/get-started/install-the-hyper-v-role-on-windows-server) with management tools
+* Install [Docker](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server)
+* [Install OVS](http://docs.openvswitch.org/en/latest/intro/install/windows/)
+and configure the daemons as Windows service.
+  * The kernel driver of OVS should be [signed by Windows Hardware Dev Center](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/driver-signing)
+  * If OVS driver is not signed, please refer to the Windows doc about how to [install a test-signed driver ackage on the test computer](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/installing-a-test-signed-driver-package-on-the-test-computer).
+  *  If you don't have a self-signed OVS package and just want to try the Antrea on windows, Antrea provides a test-signed OVS package for you. See details in [Join Windows worker nodes](#Join-Windows-worker-nodes) section.
+
+### Installation
+#### Download & configure Antrea for Linux
+Configure the Antrea for Linux on master node following [Getting started](/docs/getting-started.md) document.
+```
+# Example:
+kubectl apply -f  https://raw.githubusercontent.com/ruicao93/antrea/windows_install/build/yamls/antrea-windows.yml
+```
+#### Add Windows Antrea Agent and kube-proxy DaemonSets
+Now you can add Windows-compatible versions of Antrea Agent and kube-proxy by applying yaml files privided by Antrea:
```

Comment:
s/privided/provided

Reply:
it seems we no longer provide the kube-proxy yaml file, but instead use the upstream one, so this statement doesn't seem accurate. I'm referring to your PR https://github.com/vmware-tanzu/antrea/pull/695.

Reply:
Yes, we indeed don't provide kube-proxy.yaml in the latest change. Thanks~~

Fixed.